library/core/src/ptr/mut_ptr.rs

```diff
@@ -100,7 +100,7 @@ impl<T: ?Sized> *mut T {
     /// coercion.
```

Comment:
fix: looks like the comment here (well, line 99 where github won't let me put it directly) needs an update too -- `as_mut` to `cast_mut`.

Consider making it a ``` [`pointer::cast_mut`] ``` intra-doc link while you're there.  (That might be the wrong syntax; I'm not sure how best to do it for primitives.)